site/types/badger.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
+// @flow
+
+declare type Badger = {
```

Comment:
I was following consistency with other types in this folder. Should I refactor others?